lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -43,8 +43,13 @@ static int getMMAVersionSafe(int computeCapability, tt::DotOp op) {
   return 0;
 }
 
-SmallVector<unsigned, 2>
+SmallVector<unsigned, 3>
```

Comment:
nit: I would consider using `SmallVector<unsigned>` in all those places instead

Reply:
What about when some function returns `SmallVector<unsigned, N>`, should I change those ?

Reply:
Sounds good